RELEASE-NOTES.md

```diff
@@ -3,6 +3,7 @@
 ## PyMC3 vNext (TBD)
 ### Breaking Changes
 - ⚠ Theano-PyMC has been replaced with Aesara, so all external references to `theano`, `tt`, and `pymc3.theanof` need to be replaced with `aesara`, `aet`, and `pymc3.aesaraf` (see [4471](https://github.com/pymc-devs/pymc3/pull/4471)).
+- ⚠ The `pymc3.memoize` module was removed and replaced with `cachetools`.  The `hashable` function and `WithMemoization` class were moved to `pymc3.util`.
```

Comment:
This should be under `Maintenance` I think.

Reply:
OK, I'll change that.